packages/webpack/react-webpack-plugin/src/ReactWebpackPlugin.ts

```diff
@@ -224,6 +249,106 @@ class ReactWebpackPlugin {
             });
         },
       );
+      if (options.extractStr) {
```

Comment:
nitpick: I think it would be great if you can write these code related with extractStr to a standalone file. E.g.: `ExtractStrWebpackPlugin.ts` so that it could be tested standalone